packages/components/codemods/migrateGuidanceBlockActionsToActionsSlot/migrateGuidanceBlockActionsToActionsSlot.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { parseJsx } from '../__tests__/utils'
+import {
+  getKaioTagNamesMapByComponentName,
+  printAst,
+  transformSource,
+  type TransformSourceArgs,
+} from '../utils'
+import { migrateGuidanceBlockActionsToActionsSlot } from './migrateGuidanceBlockActionsToActionsSlot'
+
+const transformGuidanceBlock = (sourceFile: TransformSourceArgs['sourceFile']): string => {
+  const kaioTagNamesMap = getKaioTagNamesMapByComponentName(sourceFile, ['GuidanceBlock'])
+  return transformSource({
+    sourceFile,
+    transformers: [migrateGuidanceBlockActionsToActionsSlot(kaioTagNamesMap!)],
+  })
+}
+
+describe('transformActionsToButtonNext()', () => {
+  it('imports Button from the /next path and transforms all button-like actions prop into a Button component', () => {
```

Comment:
Would be good to add in a tooltip one that shows the expected non behaviour of this codemod for the older tooltips